src/caustics/cosmology.py

```diff
@@ -420,14 +420,7 @@ def comoving_distance(
         """
         Ode0 = 1 - Om0
         ratio = (Om0 / Ode0) ** (1 / 3)
-        return (
-            self.hubble_distance(h0)
-            * (
-                self._comoving_distance_helper((1 + z) * ratio, params)
-                - self._comoving_distance_helper(ratio, params)
-            )
-            / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))
-        )
+        return self.hubble_distance(h0) * (self._comoving_distance_helper((1 + z) * ratio, params) - self._comoving_distance_helper(ratio, params)) / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))  # fmt: skip
```

Comment:
In order to make this line more readable, I would rdefine new variables for the comoving distance primitive calculation above:

```
Ode0 = 1 - Om0
ratio = (Om0 / Ode0) ** (1 / 3)
H0 = self.hubble_distance(h0)
# fmt: off
D1 = self._comoving_distance_helper((1 + z) * ratio, params) # comoving distance primitive at z
D0 = self._comoving_distance_helper(ratio, params) # comoving distance primitive at z=0
# fmt: on
return H0 * (D1 - D0) / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))  # fmt: skip
```